README.md

```diff
@@ -162,6 +169,26 @@ CANDIG_OPA_SITE_ADMIN_KEY=
 INSIDE_CANDIG=
 ```
 
+## Standalone PostGres db and AdMiner
+
+For local development, you can quickly deploy a local database server (postgres) and management tool (adminer) with `docker-compose`. Make sure your postgres env variables are set in `.env` before running:
+
+```
+# Start docker-compose containers
+docker-compose -f docker-compose.dev.yaml up -d
```

Comment:
bento generally uses `docker compose`, not `docker-compose` + `docker-compose` (non-plugin form) is deprecated afaik.